cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/ScheduleSpecificationCodecTest.java

```diff
@@ -75,8 +80,8 @@ public void testBackwardsCompatibility() throws Exception {
     // Before 2.8, the ScheduleSpecificationCodec used to have the same behavior as what Gson would do by
     // default, and only Schedules existed. We make sure that ScheduleSpecification persisted before
     // 2.8 can still be deserialized using the new codec.
```

Comment:
yeah... the way schedules are serialized/deserialized is weird.  I will open a jira for this.